modelling/src/main/java/org/axonframework/modelling/entity/child/AbstractEntityChildModel.java

```diff
@@ -142,35 +141,47 @@ protected Builder(@Nonnull Class<P> parentClass,
         }
 
         /**
-         * Sets the {@link ChildEntityMatcher} to use for matching the child entities to the command. This is used to
-         * filter the child entities based on the command. One entity must match an incoming command. If no match is
-         * found, an {@link ChildEntityNotFoundException} is thrown. If multiple matches are found, an
-         * {@link ChildAmbiguityException} is thrown.
+         * Sets the {@link CommandTargetResolver} to use for resolving the child entity to handle the command. This
+         * should return one child entity, or {@code null} if no child entity should handle the command.
+         * <p>
+         * Defaults to matching a singular child entity, or throwing a {@link ChildAmbiguityException} if more than one
```

Comment:
I don't think this class has a default. Only the `SingleEntityChildModel` has a default.

Reply:
Good catch